app/models/student_loans.rb

```diff
@@ -81,6 +82,10 @@ def first_eligible_qts_award_year(claim_year = nil)
     ].max
   end
 
+  def last_eligible_qts_award_year
+    POLICY_END_YEAR
+  end
```

Comment:
Nice, this method could also be used to generate [this](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/71dcdb5cc27de088b77519a20b6090f4a3778def/config/locales/en.yml#L276) text but probably not worth the effort.